framework/resources/Datasources/ctas/DRILL-1248/ctas.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+source conf/drillTestConfig.properties
+
+hadoop fs -mkdir -p /drill/testdata/DRILL-1248
```

Comment:
Can we please delete the directory first, to prevent corruption during reruns?

Reply:
I will not delete a source directories. This folder may be used by other test suites in the future, and they may be corrupted by multithreading.

Reply:
So for that, please use the 'cp' command in the test framework - it'll take care of copying over source data. The gen script should be re-runnable without failures. Here, it will fail in the below copy step.

Reply:
Done.